docs/whatisk8s.md

```diff
@@ -0,0 +1,40 @@
+
+# What is Kubernetes?
+
+Kubernetes is an open source container cluster orchestration platform that was started by Google in 2014.
```

Comment:
1. I think the term "orchestration" does us a disservice. We should avoid it.
2. "open source container cluster orchestration" is an unhyphenated run-on adjective.
3. The project was started in 2014, not the platform.

The Kubernetes project was started by Google in 2014. Kubernetes is an open-source platform for automating deployment, scaling, and operations of application containers. It schedules containers across clusters of hosts.